cmd/cli/install.go

```diff
@@ -190,6 +193,7 @@ func newInstallCmd(config *helm.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVar(&inst.deployJaeger, "deploy-jaeger", defaultDeployJaeger, "Deploy Jaeger in the namespace of the OSM controller")
 	f.StringVar(&inst.envoyLogLevel, "envoy-log-level", defaultEnvoyLogLevel,
 		fmt.Sprintf("Envoy log level is the level of logs collected from envoy and needs to be one of: %s", strings.Join(possibleEnvoyLogLevels, ",")))
+	f.StringVar(&inst.envoyImage, "envoy-image", defaultEnvoyImage, "Envoy image is the Envoy proxy sidecare image")
```

Comment:
We should not be adding new CLI options here going forward, we have a pending issue to use `--set`: https://github.com/openservicemesh/osm/issues/3124